kentik_api_library/setup.py

```diff
@@ -260,7 +263,7 @@ def run(self):
         "requests[socks]>=2.25.0",
```

Comment:
It would be really useful is to figure out a way to remove duplication of package version specifications in `requirements.txt` and in `setup.py`. That would be a RealBenefit(tm).

Reply:
@mateuszmidor Could you take a look at this? Maybe we should create a ticket to investigate this